infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationManager.java

```diff
@@ -58,8 +58,8 @@ public CorfuReplicationManager(LogReplicationContext context, NodeDescriptor loc
      * Start Log Replication Manager, this will initiate a runtime against
      * each Sink session(cluster + replication model + client), to further start log replication.
      */
-    public void start() {
-        for (ClusterDescriptor remoteCluster : context.getTopology().getSinkClusters().values()) {
+    public void start(Map<String, ClusterDescriptor> connectionEndpoints) {
```

Comment:
Would you like to update the comment to reflect the input parameter?

Reply:
this is not needed when rebased